examples/storage/cassandra/README.md

```diff
@@ -215,6 +215,10 @@ spec:
           capabilities:
             add:
               - IPC_LOCK
+        lifecycle:
```

Comment:
@saad-ali @rootfs I think the `mungedocs` scripts may be responsible for updating the docs.